pm/recipient.go

```diff
@@ -45,7 +45,7 @@ type Recipient interface {
 	TxCostMultiplier(sender ethcommon.Address) (*big.Rat, error)
 
 	// MinEV returns the recipients EV requirement as configured on startup
```

Comment:
Comment needs to be updated to reflect new method name

Reply:
Fixed in https://github.com/livepeer/go-livepeer/pull/956/commits/789195926a3d4628b3bc3e6acec20f4b78035e68